### ISSUE TITLE
lime labels differ from Keras model predictions

### ISSUE THREAD

**Issue by giuliaass** [marcotcr/lime#578]
I'm working on a Text classification Problem with Keras and lime. I'm using neural networks and word embeddings. 

Afterwards I want to explain my model with lime.
When I use lime to explain the first three sentences of the test data lime predicts for me the following labels: 1 0 1 1 0 1. But my Keras model predicts: 1 0 1 1 1 1
Also the probabilities for label 1 or label 0 differ for the Keras model and the lime output.

Why is this the case or is there something wrong with my model?

Thank you in advance!

**Comment by marcotcr** [comment]
Make sure your model takes in a list rather than a single example. If it doesn't work, please open a new issue with a code sample.
